doc: add release notes for 1.7.0 (#5534)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-1.7.0.rst

.. _ncs_release_notes_changelog:

|NCS| v1.6.0 Release Notes
##########################

.. contents::
   :local:
   :depth: 2

|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.

The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
nRF53 Series devices are now supported for production.

> Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.

Author: Hi, @jdstapes , do these lines need to be updated? Checking as per https://projecttools.nordicsemi.no/jira/browse/NCSDK-11021 -- "Check/update the development/production support statement". Thanks!

Author: Checked at the doc review meeting. No changes needed.

* Added configuration for nRF52840 DK that supports data forwarder over NUS to :ref:`nrf_machine_learning_app`.

Pelion
------

* Updated Pelion Device Management Client library version to 4.10.0.
* Switched to using :ref:`caf_power_manager` and :ref:`caf_net_state` in :ref:`pelion_client`.

MCUboot
=======

The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``7a51968``, plus some |NCS| specific additions.

The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.

The following list summarizes the most important changes inherited from upstream MCUboot:

* Added AES support for image encryption (based on mbedTLS).
* MCUboot serial: Ported encoding to use the cddl-gen module (which removes dependency on the `TinyCBOR`_ library).
* bootutil_public library: Made ``boot_read_swap_state()`` declaration public.

Mcumgr
======

The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``74e77ad08``.

The following list summarizes the most important changes inherited from upstream mcumgr:

> * No changes yet

Author: @nvlsianpu , @de-nordic , should we add any updates here?

Author: Edits provided by @de-nordic , thanks!

.. code-block:: none

   git log --oneline manifest-rev ^14f09a3b00

The current |NCS| master branch is based on the Zephyr v2.7 development branch.

Matter (Project CHIP)
=====================

The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``b77bfb047374b7013dbdf688f542b9326842a39e``.

The following list summarizes the most important changes inherited from the upstream Matter:

* Added:

  * Support for Certificate-Authenticated Session Establishment (CASE) for communication among operational Matter nodes.
  * Support for OpenThread's DNS Client to enable Matter node discovery on Thread devices.

* Updated:

  * Fixed the known issue KRKNWK-10387 where Matter service was needlessly advertised over Bluetooth® LE during DFU.
    Now if Matter pairing mode is not opened and the Bluetooth® LE advertising is needed due to DFU requirements, only the SMP service is advertised.

Partition Manager
=================

* Reworked how external flash memory support is enabled.
  The MCUboot secondary partition can now be placed in external flash memory without modifying any |NCS| files.

> Documentation

Author: WIP

Author: Updated.

   :local:
   :depth: 2

|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.

The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
nRF53 Series devices are now supported for production.
Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
See the release notes and documentation for those protocols and libraries for further information.

Highlights
**********

* Software support for :ref:`zephyr:thingy53_nrf5340`:

  * :ref:`nrf_machine_learning_app` application with Edge Impulse ML development platform, supported for development.
  * :ref:`Matter weather station <matter_weather_station_app>` application
  * Bluetooth® samples:

    * :ref:`central_uart`
    * :ref:`peripheral_lbs`
    * :ref:`peripheral_uart`

  * Bluetooth mesh samples:

>     * :ref:`bluetooth_mesh_light`

Reviewer: Thingy:53 is not added for all these mesh samples
Checking with PMT if all should have.

Author: Removed three out of five samples after receiving an update from @Balaklaka

    * :ref:`bluetooth_mesh_light_switch`
    * :ref:`bluetooth_mesh_light_lc`
    * :ref:`bluetooth_mesh_sensor_server`
    * :ref:`bluetooth_mesh_sensor_client`

  * Thread samples:

    * :ref:`coap_client_sample`
    * :ref:`coap_server_sample`

* Multi-image DFU for nRF53 Series with simultaneous application and network core firmware update.
* :ref:`ug_radio_fem` for the nRF21540 FEM range extender for product development.

  * :ref:`direct_test_mode` and :ref:`radio_test` sample support.
  * Application support for protocols based on Bluetooth® LE and 802.15.4 (Thread and Zigbee), using :ref:`nrfxlib:mpsl`.
  * Support for :ref:`nRF21540 development kit (DK) <nrf21540dk_nrf52840>`, :ref:`nRF21540 evaluation kit (EK) <ug_radio_fem_nrf21540_ek>` and custom board configurations.

* Added Wi-Fi Coex feature supported for development.
* Added support for NFC and *pair before use* type of accessories to the Apple Find My add-on.

.. note::
    Programming nRF52832 revision 3 and nRF52840 revision 3 devices requires nrfjprog version 10.13 or newer.
    nrfjprog is part of the `nRF Command Line Tools`_.

> 

Reviewer 2: ```suggestion

* nRF Connect for Cloud is now nRF Cloud.

```

    When **Button 1** was pressed and released while holding one of the other buttons, the notification for release was the same as for press.
  * Fixed an issue in the :ref:`gatt_dm_readme` library where a memory fault could happen if a peer device disconnected during the service discovery process.
  * :ref:`lbs_readme` library - Added write request data validation in the LED characteristic.

Bluetooth mesh
---------------

* Added:

  * The choice of default Bluetooth® LE Controller changed from Zephyr Bluetooth® LE Controller to SoftDevice Controller.
  * Bluetooth® mesh profiles and models are ready for production.

* Updated:

  * Updated the :ref:`bt_mesh_light_hsl_srv_readme` and the :ref:`bt_mesh_light_xyl_srv_readme` models to no longer extend the :ref:`bt_mesh_lightness_srv_readme` model, and instead get a pointer to this model in the initialization macro.
  * Updated samples with support for the :ref:`zephyr:thingy53_nrf5340`.
  * Fixed an issue where beacons were stopped being sent after node reset.
  * Fixed an issue where the IV update procedure could be started immediately after the device has been provisioned.

>   * Fixed multiple issues in the :ref:`bt_mesh_sensor_types_readme` module.

Reviewer: Need to add migration details for mesh

Author: Please add as a comment here or a PR to this PR.

Reviewer: Waiting for this PR to get merged first: https://github.com/nrfconnect/sdk-nrf/pull/5156

Author: Migration now added.

Matter
------

* Added:

  * :ref:`Thingy:53 weather station <matter_weather_station_app>` application.
  * :ref:`Template <matter_template_sample>` sample with a guide about :ref:`ug_matter_creating_accessory`.
  * :ref:`ug_matter_tools` page with information about building options for Matter controllers.
  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Matter.
  * :doc:`Matter documentation set <matter:index>` based on the documentation from the Matter submodule.

Thread
------

* Added:

>   * Added support for nRF21540 supported for nRF52 and nRF53 families, including Bluetooth® LE in the multiprotocol configuration.

Reviewer 3: Added support for nRF21540 supported ... reads kind of strange. can we somehow rephrase it?

The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
nRF53 Series devices are now supported for production.
Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
See the release notes and documentation for those protocols and libraries for further information.

Highlights
**********

* Software support for :ref:`zephyr:thingy53_nrf5340`:

  * :ref:`nrf_machine_learning_app` application with Edge Impulse ML development platform, supported for development.
  * :ref:`Matter weather station <matter_weather_station_app>` application
  * Bluetooth® samples:

    * :ref:`central_uart`
    * :ref:`peripheral_lbs`
    * :ref:`peripheral_uart`

  * Bluetooth mesh samples:

    * :ref:`bluetooth_mesh_light`
    * :ref:`bluetooth_mesh_light_switch`
    * :ref:`bluetooth_mesh_light_lc`
    * :ref:`bluetooth_mesh_sensor_server`
    * :ref:`bluetooth_mesh_sensor_client`

>   * Thread samples:

Reviewer 4: Thread examples are not supported for Thingy.

    * :ref:`coap_client_sample`
    * :ref:`coap_server_sample`

* Multi-image DFU for nRF53 Series with simultaneous application and network core firmware update.
* :ref:`ug_radio_fem` for the nRF21540 FEM range extender for product development.

  * :ref:`direct_test_mode` and :ref:`radio_test` sample support.
  * Application support for protocols based on Bluetooth® LE and 802.15.4 (Thread and Zigbee), using :ref:`nrfxlib:mpsl`.
  * Support for :ref:`nRF21540 development kit (DK) <nrf21540dk_nrf52840>`, :ref:`nRF21540 evaluation kit (EK) <ug_radio_fem_nrf21540_ek>` and custom board configurations.

> * Added Wi-Fi Coex feature supported for development.

Reviewer 5: This should specify for which protocol stack. I.e. 802.15.4 (@czeslawmakarski).

The following list summarizes the most important changes inherited from the upstream Matter:

* Added:

  * Support for Certificate-Authenticated Session Establishment (CASE) for communication among operational Matter nodes.
  * Support for OpenThread's DNS Client to enable Matter node discovery on Thread devices.

* Updated:

  * Fixed the known issue KRKNWK-10387 where Matter service was needlessly advertised over Bluetooth® LE during DFU.
    Now if Matter pairing mode is not opened and the Bluetooth® LE advertising is needed due to DFU requirements, only the SMP service is advertised.

Partition Manager
=================

* Reworked how external flash memory support is enabled.
  The MCUboot secondary partition can now be placed in external flash memory without modifying any |NCS| files.

Documentation
=============

In addition to documentation related to the changes listed above, the following documentation has been updated:

* General changes:

  * Updated the style and formatting of all figures across all documentation pages.

>   * Added new documentation sets for :ref:`Trusted Firmare-M <tfm:index>` and :ref:`Matter <matter:index>`.

Reviewer: It seems like this line is causing the doc build to fail...

It includes the TF-M, MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.

The |NCS| is where you begin building low-power wireless applications with Nordic Semiconductor nRF52, nRF53, and nRF91 Series devices.
nRF53 Series devices are now supported for production.
Wireless protocol stacks and libraries may indicate support for development or support for production for different series of devices based on verification and certification status.
See the release notes and documentation for those protocols and libraries for further information.

Highlights
**********

* Software support for :ref:`zephyr:thingy53_nrf5340`:

  * :ref:`nrf_machine_learning_app` application with Edge Impulse ML development platform, supported for development.
  * :ref:`Matter weather station <matter_weather_station_app>` application
  * Bluetooth® samples:

    * :ref:`central_uart`
    * :ref:`peripheral_lbs`
    * :ref:`peripheral_uart`

  * Bluetooth mesh samples:

    * :ref:`bluetooth_mesh_light`
    * :ref:`bluetooth_mesh_light_switch`

* Multi-image DFU for nRF53 Series with simultaneous application and network core firmware update.

> * :ref:`ug_radio_fem` for the nRF21540 FEM range extender for product development.

Reviewer 6: Change to: ":ref:`ug_radio_fem` for the nRF21540 FEM range extender is now supported for production."

  * :ref:`direct_test_mode` and :ref:`radio_test` sample support.

>   * Application support for protocols based on Bluetooth® LE and 802.15.4 (Thread and Zigbee), using :ref:`nrfxlib:mpsl`.

Reviewer 6: Change to: "Support for Bluetooth® LE, Thread, Zigbee and multiprotocol applications and samples using :ref:`nrfxlib:mpsl`."

  * Support for :ref:`nRF21540 development kit (DK) <nrf21540dk_nrf52840>`, :ref:`nRF21540 evaluation kit (EK) <ug_radio_fem_nrf21540_ek>` and custom board configurations.

* Added Wi-Fi Coex feature for the 802.15.4 radio driver, supported for development.
* Added support for NFC and *pair before use* type of accessories to the Apple Find My add-on.
* nRF Connect for Cloud is now nRF Cloud.
* Updated the required :ref:`minimum CMake version <gs_recommended_versions>` to 3.20.0.

> 

Reviewer 6: Add this bullet: "Added support for nRF Connect SDK development in nRF Connect for Visual Studio Code"

> 

Reviewer 6: Add this bullet: "Added production support for nRF52833 for Bluetooth LE HomeKit accessories."

  * :ref:`ug_matter_tools` page with information about building options for Matter controllers.
  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Matter.
  * :doc:`Matter documentation set <matter:index>` based on the documentation from the Matter submodule.

Thread
------

* Added:

  * Added support for nRF21540 for nRF52 Series and nRF53 Series, including Bluetooth® LE in the multiprotocol configuration.
  * Improvements in :ref:`thread_ug_supported_features_v12`:

    * Thread 1.2 supported in all samples.
    * Retransmissions are now supported when transmission security is handled by the radio driver.
    * Support for CSL Accuracy TLV in the MLE Parent Response.
    * Link Metrics data is now properly updated when using ACK-based Probing.

  * Added support for Thread Backbone Border Router on the :ref:`thread_architectures_designs_cp_rcp` architecture.

* Updated:

  * :ref:`ot_cli_sample` sample - Updated with the USB support.
  * :ref:`ot_coprocessor_sample` sample - Updated with the USB support.
  * Removed ``NET_SHELL`` from the Thread samples due to its limited usefulness.

Zigbee
------

> 

Reviewer 6: Add this information: "In this release, Zigbee is supported for development and should not be used for production. nRF Connect SDK v1.5.1 contains the certified Zigbee solution supported for production."

* Added:

  * Added production support for :ref:`radio front-end module (FEM) <ug_radio_fem>` for nRF52 Series devices and nRF21540 EK.
  * Added development support for :ref:`radio front-end module (FEM) <ug_radio_fem>` for nRF53 Series devices and nRF21540 EK.
  * Added development support for ``nrf5340dk_nrf5340_cpuapp`` to the :ref:`zigbee_ncp_sample` sample.
  * :ref:`lib_zigbee_zcl_scenes` library with documentation.
    This library was separated from the Zigbee light bulb sample.
  * :ref:`zigbee_template_sample` sample.
    This minimal Zigbee router application can be used as the starting point for developing custom Zigbee devices.
  * Added API for vendor-specific NCP commands.
    See the :ref:`Zigbee NCP sample <zigbee_ncp_vendor_specific_commands>` page for more information.
  * Added API for Zigbee command for getting active nodes.

* Updated:

  * ZBOSS Zigbee stack to version 3.8.0.1+4.0.0.
    See the :ref:`nrfxlib:zboss_changelog` in the nrfxlib documentation for detailed information.

>   * :ref:`ug_zigbee_tools_ncp_host` to the production-ready v1.0.0.

Reviewer 6: Change to: ":ref:`ug_zigbee_tools_ncp_host` is now supported for production and moved to version 1.0.0."

  * :ref:`zigbee_ug_logging_stack_logs` - Improved printing ZBOSS stack logs.
    Added new backend options to print ZBOSS stack logs with option for using binary format.
  * Fixed the KRKNWK-9743 known issue where the timer could not be stopped in Zigbee routers and coordinators.
  * Fixed the KRKNWK-10490 known issue that would cause a deadlock in the NCP frame fragmentation logic.
  * Fixed the KRKNWK-6071 known issue with inaccurate ZBOSS alarms.
  * Fixed the KRKNWK-5535 known issue where the device would assert if flooded with multiple Network Address requests.
  * Fixed an issue where the NCS would assert in the host application when the host started just after SoC's SysReset.

ESB
---

* Updated:

  * Modified the ESB interrupts configuration to reduce the ISR latency and enable scheduling decision in the interrupt context.

nRF IEEE 802.15.4 radio driver
------------------------------

* Added:

  * :ref:`802154_phy_test` sample, with an experimental Antenna Diversity functionality.

>   * Experimental Wi-Fi Coexistence functionality.

Reviewer 6: Change to: "Wi-Fi coexistence functionality supported for development."

    Now if Matter pairing mode is not opened and the Bluetooth® LE advertising is needed due to DFU requirements, only the SMP service is advertised.

Partition Manager
=================

* Reworked how external flash memory support is enabled.
  The MCUboot secondary partition can now be placed in external flash memory without modifying any |NCS| files.

Documentation
=============

In addition to documentation related to the changes listed above, the following documentation has been updated:

* General changes:

  * Added cross-search functionality to the documentation search feature available at the top-left corner of each documentation page.
    Searching now parses all :ref:`documentation sets <ncs_introduction>` pages and displays the results for each set.
    For example, results from the :ref:`nrfxlib:nrfxlib` documentation set will be listed with ``nrfxlib >>`` before the page title.
  * Updated the pages in the :ref:`getting_started` section with information about the support for the new Visual Studio Code extension.
  * Updated the style and formatting of all figures across all documentation pages.

>   * Added new documentation sets for `Trusted Firmare-M <TF-M documentation_>` and :doc:`Matter <matter:index>`.

Reviewer 7: Link not working
```suggestion
  * Added new documentation sets for `Trusted Firmare-M <TF-M documentation_>`_ and :doc:`Matter <matter:index>`.
```

    Now if Matter pairing mode is not opened and the Bluetooth® LE advertising is needed due to DFU requirements, only the SMP service is advertised.

Partition Manager
=================

* Reworked how external flash memory support is enabled.
  The MCUboot secondary partition can now be placed in external flash memory without modifying any |NCS| files.

Documentation
=============

In addition to documentation related to the changes listed above, the following documentation has been updated:

* General changes:

  * Added cross-search functionality to the documentation search feature available at the top-left corner of each documentation page.
    Searching now parses all :ref:`documentation sets <ncs_introduction>` pages and displays the results for each set.
    For example, results from the :ref:`nrfxlib:nrfxlib` documentation set will be listed with ``nrfxlib >>`` before the page title.
  * Updated the pages in the :ref:`getting_started` section with information about the support for the new Visual Studio Code extension.
  * Updated the style and formatting of all figures across all documentation pages.

>   * Added new documentation sets for :ref:`Trusted Firmare-M <TF-M documentation_>` and :doc:`Matter <matter:index>`.

Reviewer 7: ```suggestion
  * Added new documentation sets for `Trusted Firmare-M <TF-M documentation_>`_ and :doc:`Matter <matter:index>`.
```

  * :ref:`getting_started` section pages:

    * :ref:`gs_recommended_versions` - Now includes information about the supported operating systems, previously listed on a separate page.
    * :ref:`gs_assistant` - Added information about Toolchain manager being available for macOS and steps for building from SEGGER Embedded Studio.
    * :ref:`gs_installing` - Added information about the version folder created when extracting the GNU Arm Embedded Toolchain and applied minor fixes based on usability testing (for example, to the Linux installation instructions).

  * :ref:`ug_app_dev` section pages:

    * :ref:`ug_multi_image` - Updated with the section about Child image devicetree overlays.
    * :ref:`ug_radio_fem` - Updated for the nRF21540 release.

  * :ref:`ug_nrf91` user guide - Restructured into several subpages.
  * :ref:`ug_nrf5340` user guide - Reworked with major updates.
  * :ref:`protocols` section pages:

    * :ref:`ug_matter` pages:

      * :ref:`ug_matter_tools` - New page.
      * :ref:`ug_matter_creating_accessory` - New page.

    * :ref:`ug_thread` pages:

      * :ref:`ug_thread_configuring` - Updated with new configuration options.

>       * :ref:`ug_thread_supported_features` - Updated with new supported features.

Reviewer 7: ```suggestion
      * :ref:`thread_ug_supported_features` - Updated with new supported features.
```

    * :ref:`zigbee_samples`:

      * :ref:`zigbee_ncp_sample` - Updated with information about logging ZBOSS traces and about vendor-specific commands.

  * :ref:`libraries` section pages:

    * :ref:`profiler` - Several updates.

  * :ref:`documentation` section pages:

    * :ref:`doc_build` - Updated the documentation building procedure.
    * :ref:`doc_styleguide` - Simplified the guidelines.

  * Renamed :ref:`ncs_release_notes_changelog` (this page).

nrfxlib
-------

* :ref:`nrfxlib:mpsl` section pages:

  * :ref:`nrfxlib:mpsl_clock` - Page added.
  * :ref:`nrfxlib:mpsl_cx` - Page added.
  * :ref:`nrfxlib:mpsl_fem` - Updated the nRF21540 usage section.
  * :ref:`nrfxlib:mpsl_lib` - Updated different parts of the entire page.

* :ref:`nrfxlib:nrf_802154` section pages:

  * :ref:`nrfxlib:radiodriver_api` - Page added.
  * :ref:`nrfxlib:rd_feature_description` - Updated the Performing retransmissions section.

* :ref:`nrf_modem` section pages:

  * :ref:`nrfxlib:nrf_modem_api` - Updated with new API sections.
  * :ref:`full_dfu` - Updated with major changes.

>   * Removed the "AT commands" page.
>   * :ref:`nrfxlib:nrf_modem_at` - Page added.
>   * :ref:`nrfxlib:at_socket` - Page added.

Reviewer 7: ```suggestion
  * Renamed the "AT commands" page to :ref:`at_socket`
  * :ref:`nrfxlib:nrf_modem_at` - Page added.
```

Other samples
-------------

* :ref:`radio_test` - Added an automatic build of the :ref:`nrf5340_empty_app_core` sample, when building for ``nrf5340dk_nrf5340_cpunet``.

Common
======

The following changes are relevant for all device families.

sdk-nrfxlib
-----------

* Updated the default :ref:`nrf_rpc` transport backend to use the RPMsg Service library.

See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.

Modem library
+++++++++++++

* Added:

  * Added a new API for AT commands.
    See :ref:`nrfxlib:nrf_modem_at` for more information.
  * Added a new API for modem delta firmware updates.
    See :ref:`nrfxlib:nrf_modem_delta_dfu` for more information.

* Updated:

  * Updated :ref:`nrf_modem` to version 1.3.0.
    See the :ref:`nrfxlib:nrf_modem_changelog` for detailed information.

* Deprecated:

  * The AT socket API is now deprecated.
  * The DFU socket API is now deprecated.

> 

Author: Added, please check.

* Updated:

  * ZBOSS Zigbee stack to version 3.8.0.1+4.0.0.
    See the :ref:`nrfxlib:zboss_changelog` in the nrfxlib documentation for detailed information.
  * :ref:`ug_zigbee_tools_ncp_host` is now supported for production and updated to version 1.0.0.
  * :ref:`zigbee_ug_logging_stack_logs` - Improved printing ZBOSS stack logs.
    Added new backend options to print ZBOSS stack logs with option for using binary format.
  * Fixed the KRKNWK-9743 known issue where the timer could not be stopped in Zigbee routers and coordinators.
  * Fixed the KRKNWK-10490 known issue that would cause a deadlock in the NCP frame fragmentation logic.
  * Fixed the KRKNWK-6071 known issue with inaccurate ZBOSS alarms.
  * Fixed the KRKNWK-5535 known issue where the device would assert if flooded with multiple Network Address requests.
  * Fixed an issue where the NCS would assert in the host application when the host started just after SoC's SysReset.

ESB
---

* Updated:

  * Modified the ESB interrupts configuration to reduce the ISR latency and enable scheduling decision in the interrupt context.

nRF IEEE 802.15.4 radio driver
------------------------------

* Added:

> 

Reviewer 8: ```suggestion
  * Updated ``NRF_802154_SWI_PRIORITY``, ``NRF_802154_ECB_PRIORITY`` and ``NRF_802154_SL_RTC_IRQ_PRIORITY`` to comply with the limitation introduced in :ref:`nrfxlib:nrf_802154_driver`.
```

Not sure if the syntax is correct, but this entry is related with https://github.com/nrfconnect/sdk-nrf/pull/5640.

Common Application Framework (CAF)
----------------------------------

* Added :ref:`caf_net_state`.
* Added :ref:`caf_power_manager`.
* Updated :ref:`caf_sensor_sampler` with a limit to the number of ``sensor_event`` events that it submits.

Profiler
--------

* Added profiling string data.
* Optimized numeric data encoding.

Edge Impulse
------------

* Added support for Thingy:53 to :ref:`nrf_machine_learning_app`.
* Added configuration for nRF52840 DK that supports data forwarder over NUS to :ref:`nrf_machine_learning_app`.

Pelion
------

* Updated Pelion Device Management Client library version to 4.10.0.
* Switched to using :ref:`caf_power_manager` and :ref:`caf_net_state` in :ref:`pelion_client`.
* Updated the :ref:`application documentation <pelion_client>` with a step that requires downloading Pelion development tools.

Build system
------------

* Bugfixes:

  * Fixed a bug where :file:`dfu_application.zip` would not be updated after rebuilding the code with changes.

MCUboot
=======

> The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``7a51968``, plus some |NCS| specific additions.

Author: @sigvartmh , @hakonfam , can you confirm this is the correct commit ID?

Reviewer 9: @SebastianBoe to confirm

Reviewer 10: Was the wrong commit. Fixed now.

The code for integrating MCUboot into |NCS| is located in :file:`ncs/nrf/modules/mcuboot`.

The following list summarizes the most important changes inherited from upstream MCUboot:

* Added AES support for image encryption (based on mbedTLS).
* MCUboot serial: Ported encoding to use the cddl-gen module (which removes dependency on the `TinyCBOR`_ library).
* bootutil_public library: Made ``boot_read_swap_state()`` declaration public.

Mcumgr
======

> The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``e64f5060b``.

Author: @de-nordic , please confirm this is the correct commit ID.

Author: Updated ``657deb65`` after checking on Teams.

The following list summarizes the most important changes inherited from upstream mcumgr:

* Fixed an issue with SMP file management commands that would fail to read or write files, or both.

Zephyr
======

.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.

> The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``14f09a3b00``, plus some |NCS| specific additions.

Author: @mbolivar-nordic , can you confirm this is the correct commit ID? One last check before merge.
